src/jobflow/settings.py

```diff
@@ -136,10 +137,21 @@ def load_default_settings(cls, values):
         from monty.serialization import loadfn
 
         config_file_path: str = values.get("CONFIG_FILE", DEFAULT_CONFIG_FILE_PATH)
-
         new_values = {}
         if Path(config_file_path).exists():
-            new_values.update(loadfn(config_file_path))
+            if Path(config_file_path).stat().st_size == 0:
+                warnings.warn(
+                    f"An empty JobFlow config file was located at {config_file_path}"
+                )
+            else:
+                try:
+                    new_values.update(loadfn(config_file_path))
+                except ValidationError:
+                    raise ValueError(
```

Comment:
Strangely codecov is is saying this that this line isn't being tested. Do you have any ideas?

Could this be because pydantic is actually raising a `ValueError` to begin with, so this except is not being hit?

Reply:
Haha - I saw this last night and couldn't figure it out, but I looked again and I think you're right. It's confusing because the printed stacktrace says `ValidationError`:

```
pydantic/env_settings.py:40: in pydantic.env_settings.BaseSettings.__init__
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   ???
E   pydantic.error_wrappers.ValidationError: 1 validation error for JobflowSettings
E   __root__
E     dictionary update sequence element #0 has length 1; 2 is required (type=value_error)

pydantic/main.py:341: ValidationError
```

But the error you have to catch is `ValueError`.

Anyway, I switched to `ValueError` and made an assertion about the message.